shotgun_api3/shotgun.py

```diff
@@ -1473,10 +1497,13 @@ def update_project_last_accessed(self, project, user=None):
         record = self._call_rpc("update_project_last_accessed_by_current_user", params)
         result = self._parse_records(record)[0]
 
-
-    def _get_session_token(self):
-        """Hack to authenticate in order to download protected content
-        like Attachments
+    def get_session_token(self):
```

Comment:
The original comments dissapeared, but method is still important even tough we are not constantly refreshing the token anymore in Toolkit because when you login using username/password then you need to be able to generate the token that first time.